clang/lib/DPCT/TypeLocRewriters.cpp

```diff
@@ -45,6 +45,8 @@ class CheckTemplateArgCount {
   }
 };
 
+// This useful to print a templated type, if explicit template argument number
```

Comment:
Print a templated type. Pass a STR("") as a template argument for types with no template argument e.g. MyType<>

Reply:
done